docs/prms/prms.rst

```diff
@@ -0,0 +1,44 @@
+Pathway Reconstruction Methods
```

Comment:
This is another area where we'll need to update the contribution guide if the list of algorithms needs to be extended manually.